### PR TITLE
fix: entity list works with new input configs

### PR DESCRIPTION
**Pull request by TheSlimvReal**
closes: #XX

### Visible/Frontend Changes
- [x] 
- [ ] 

### Architectural/Backend Changes
- [x] 
- [ ]

### CODE REVIEW

File: src/app/core/entity-list/entity-list/entity-list.component.ts [hunk 1]
```diff
@@ -108,12 +112,13 @@ export class EntityListComponent<T extends Entity>
 
   isDesktop: boolean;
 
-  listName = "";
-  columns: (FormFieldConfig | string)[] = [];
-  columnGroups: GroupConfig[] = [];
+  @Input() title = "";
+  @Input() columns: (FormFieldConfig | string)[] = [];
+  @Input() columnGroups: ColumnGroupsConfig;
+  groups: GroupConfig[] = [];
```
Comment (TheSlimvReal) [inline src/app/core/entity-list/entity-list/entity-list.component.ts:118]: I am not too happy with this naming change but I did not want to require a migration

Comment (sleidig) [reply src/app/core/entity-list/entity-list/entity-list.component.ts:118]: maybe we can collect (or then do a detailed review of) all config namings as part of the Admin UI work and then prepare one large, automatic migration to clean all of this up and make it fully consistent across different components.

**Review by sleidig — APPROVED** [review]
Seems to work well now.

Thanks a lot for fixing this so quickly!

File: src/app/core/entity-list/entity-list/entity-list.component.ts (hunk 1 above)
Comment (TheSlimvReal) [reply src/app/core/entity-list/entity-list/entity-list.component.ts:121]: Lets collect them here https://github.com/Aam-Digital/ndb-core/issues/2066

**Pull request merged**